shared_kv_storage/db_store/packages/lmdb_store.cpp

```diff
@@ -397,6 +419,100 @@ tcf_err_t db_store::db_store_del(
     return db_store_del(table, inId.data(), inId.size(), inValue.data(), inValue.size());
 }
 
+// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+tcf_err_t db_store::db_store_csv_extend(
```

Comment:
How this function prevent to add duplicate value(work order id) against same worker.

Reply:
This operation will not have historical context. Rather this will be usable for extend(append/prepend) operations only for a csv formatted value.

In context with worker-work_order_id, this will be used for pending work orders only(hlding a limited number of work-order-ids always). Acting more like a queue between a producer-consumer. If a work-order with same id is submitted twice, this table will not have the intelligence to detect that as active/pending window(i.e.- the queue size) would always vary. Rather the system will rely on other storage tables(that maintain history/context) to gain that intelligence.

Reply:
Please create separte file to add csv related functions eg: lmdb_store_csv.cpp